src/wallet/bdb.cpp

```diff
@@ -145,7 +145,7 @@ bool BerkeleyEnvironment::Open(bilingual_str& err)
     LogPrintf("BerkeleyEnvironment::Open: LogDir=%s ErrorFile=%s\n", fs::PathToString(pathLogDir), fs::PathToString(pathErrorFile));
 
     unsigned int nEnvFlags = 0;
-    if (gArgs.GetBoolArg("-privdb", DEFAULT_WALLET_PRIVDB))
+    if (!m_use_shared_memory)
```

Comment:
```suggestion
    if (!m_use_shared_memory) {
```

nit (if you retouch)

Reply:
Thank you. Modified.